api/envoy/config/accesslog/v3/accesslog.proto

```diff
@@ -281,6 +285,19 @@ message GrpcStatusFilter {
   bool exclude = 2;
 }
 
+// Filters based on matching dynamic metadata.
```

Comment:
Please split this out into a separate PR. I can review that first and then we can do the other one. Thank you!

/wait